plugins/telemetry-otel/src/main/java/org/opensearch/telemetry/metrics/OTelMetricsTelemetry.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry.metrics;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.metrics.DoubleCounter;
+import io.opentelemetry.api.metrics.DoubleUpDownCounter;
+import io.opentelemetry.api.metrics.Meter;
+
+/**
+ * OTel implementation for {@link MetricsTelemetry}
+ */
+public class OTelMetricsTelemetry implements MetricsTelemetry {
+    private static final Logger logger = LogManager.getLogger(OTelMetricsTelemetry.class);
+    private final OpenTelemetry openTelemetry;
+    private final Meter otelMeter;
+
+    /**
+     * Constructor.
+     * @param openTelemetry telemetry.
+     */
+    public OTelMetricsTelemetry(OpenTelemetry openTelemetry) {
+        this.openTelemetry = openTelemetry;
+        this.otelMeter = openTelemetry.getMeter("os-meter");
```

Comment:
The instrumentation scope name should be aligned the the tracer instrumentation scope name, see please `OTelTelemetryPlugin::OTEL_TRACER_NAME`